integrations/aws/pyproject.toml

```diff
@@ -0,0 +1,104 @@
+[tool.poetry]
+name = "AWS"
+version = "0.1.0"
+description = "A short description of the project"
+authors = ["Shalev Avhar <shalev@getport.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+port_ocean = { version = "^0.5.6", extras = ["cli"] }
+boto3 = "^1.34.72"
```

Comment:
do u need all of boto?

Reply:
I couldn't find the option to install a part of a package
and I didn't see boto3 exported any extras
https://stackoverflow.com/questions/53818607/is-it-possible-to-install-part-of-python-package-via-pip
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html